backend/src/main/java/com/github/EazyErik/controller/FavouriteDrinksController.java

```diff
@@ -10,28 +10,27 @@
 import java.security.Principal;
 import java.util.List;
 
-@RestController("DrinksController")
+
+@RestController("FavouriteDrinksController")
 @RequestMapping("api/favourites")
 @RequiredArgsConstructor
 @CrossOrigin
```

Comment:
We can look into your SecurityConfig soon, so you can remove all those CrossOrigin Annotations. Not recommended for productive deployment, since request to this Controllers can come from anywhere (security problem)